mcprep_data_base.json

```
@@ -103,6 +103,25 @@
 	"Strider",
 	"Warden"
 ],
+"fixed_blocks": [
```

These aren't really fixed, so the name should reflect that. We'll say unspawnable for now, while we look at supporting Bedrock's JSON format in the future  (Minecraft is Minecraft, Java and Bedrock are equal to us)
```suggestion
"unspawnable_for_now": [
```